timApp/auth/oauth2/routes.py

```diff
@@ -45,6 +52,20 @@ def issue_token() -> Response:
     return auth_server.create_token_response()
 
 
+@oauth.get("token-validity")
```

Comment:
[TIM-koodikäytänteiden mukaan](https://tim.jyu.fi/view/tim/TIMin-kehitys/Koodikaytanteet#flask-http-polut-rest-rajapinta) HTTP-poluissa pyritään käyttämään camelCasea, eli siis

```suggestion
@oauth.get("tokenValidity")
```

Reply:
Korjattu